truss/templates/server/common/truss_server.py

```diff
@@ -213,12 +225,24 @@ def create_application(self):
             },
         )
 
+        def exit_self():
+            # Note that this kills the current process, the worker process, not
+            # the main truss_server process.
+            sys.exit()
+
+        termination_handler_middleware = TerminationHandlerMiddleware(
```

Comment:
nit: you can probably initilize this via the `middleware` kwarg. but doesn't matter.